library/src/main/java/com/novoda/downloadmanager/lib/Query.java

```diff
@@ -12,15 +15,35 @@
  * This class may be used to filter download manager queries.
  */
 public class Query {
+
+    @Retention(RetentionPolicy.SOURCE)
+    @IntDef({ORDER_ASCENDING, ORDER_DESCENDING})
+    public @interface Order {
+    }
+
     /**
      * Constant for use with {@link #orderBy}
      */
-    static final int ORDER_ASCENDING = 1;
+    public static final int ORDER_ASCENDING = 1;
 
     /**
      * Constant for use with {@link #orderBy}
      */
-    static final int ORDER_DESCENDING = 2;
+    public static final int ORDER_DESCENDING = 2;
+
+    private static final String ORDER_BY_LIVENESS = String.format("CASE batch_status "
+                    + "WHEN %1$d THEN 1 "
+                    + "WHEN %2$d THEN 2 "
+                    + "WHEN %3$d THEN 3 "
+                    + "WHEN %4$d THEN 4 "
+                    + "WHEN %5$d THEN 5 "
+                    + "ELSE 2 END",
```

Comment:
Resolves to:

``` sql
CASE batch_status 
WHEN 192 THEN 1 # 192 = STATUS_CODE_RUNNING
WHEN 190 THEN 2 # 190 = STATUS_CODE_PENDING
WHEN 193 THEN 3 # etc
WHEN 498 THEN 4 
WHEN 200 THEN 5 
ELSE 2 END
```

so this then orders the query by STATUS_CODE_RUNNING, STATUS_CODE_PENDING etc